eedl/image.py

```diff
@@ -387,9 +390,9 @@ def export(self,
 			(drive_root_folder is None or not os.path.exists(drive_root_folder)):
 
 			raise NotADirectoryError("The provided path for the Google Drive export folder is not a valid directory but"
-										" Drive export was specified. Either change the export type to use Google Cloud"
```

Comment:
Hey Adam - removing the leading spaces here will lead to the spaces being missing in the printed error message - the newlines composing the string won't get converted to spaces, in my experience. we should make sure to have spaces somewhere in here or change how we define this string